datasets/cord19/README.md

```diff
@@ -0,0 +1,226 @@
+- `url`: A `List[str]`-valued field containing all URLs associated with this paper.  Semicolon-separated.
+- `s2_id`:  A `str`-valued field containing the Semantic Scholar ID for this paper.  Can be used with the Semantic Scholar API (e.g. `s2_id=9445722` corresponds to `http://api.semanticscholar.org/corpusid:9445722`)
+
+Extra fields based on selected configuration during loading:
+
+- `fulltext`: A `str`-valued field containing the concatenation of all text sections from json (itself extracted from pdf)
+- `doc_embeddings`: A `sequence` of float-valued elements containing document embeddings as a vector of floats (parsed from string of values separated by ','). Details on the system used to extract the embeddings are available in: [SPECTER: Document-level Representation Learning using Citation-informed Transformers](https://arxiv.org/abs/2004.07180). TL;DR: it's relying on a BERT model pre-trained on document-level relatedness using the citation graph. The system can be queried through REST (see [public API documentation](https://github.com/allenai/paper-embedding-public-apis)).
+
+### Data Splits
+
+No annotation provided in this dataset so all instances are provided in training split.
+
+The sizes of each configurations are:
+
+|          | Tain   |
+| -----    | ------ |
+| metadata | 368618  |
+| fulltest | 368618  |
+| embeddings | 368618  |
+
+
+## Dataset Creation
+
+### Curation Rationale
+
+See [official readme](https://github.com/allenai/cord19/blob/master/README.md)
+
+### Source Data
+
+See [official readme](https://github.com/allenai/cord19/blob/master/README.md)
+
+#### Initial Data Collection and Normalization
+
+See [official readme](https://github.com/allenai/cord19/blob/master/README.md)
+
+#### Who are the source language producers?
+
+See [official readme](https://github.com/allenai/cord19/blob/master/README.md)
+
+### Annotations
+
+No annotations
+
+#### Annotation process
+
+N/A
+
+#### Who are the annotators?
+
+N/A
+
+### Personal and Sensitive Information
+
+[More Information Needed]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[More Information Needed]
+
+### Discussion of Biases
+
+[More Information Needed]
+
+### Other Known Limitations
+
+[More Information Needed]
+
+## Additional Information
+
+### Dataset Curators
+
+[More Information Needed]
+
+### Licensing Information
+
+[More Information Needed]
+
+### Citation Information
+
+```
+@article{Wang2020CORD19TC,
+  title={CORD-19: The Covid-19 Open Research Dataset},
+  author={Lucy Lu Wang and Kyle Lo and Yoganand Chandrasekhar and Russell Reas and Jiangjiang Yang and Darrin Eide and
+  K. Funk and Rodney Michael Kinney and Ziyang Liu and W. Merrill and P. Mooney and D. Murdick and Devvret Rishi and
+  Jerry Sheehan and Zhihong Shen and B. Stilson and A. Wade and K. Wang and Christopher Wilhelm and Boya Xie and
+  D. Raymond and Daniel S. Weld and Oren Etzioni and Sebastian Kohlmeier},
+  journal={ArXiv},
+  year={2020}
+}
+```
```

Comment:
````suggestion
```

### Contributions

Thanks to [@ggdupont](https://github.com/ggdupont) for adding this dataset.
````
we have a new section now :)